#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_octave_prune_examples.dpatch by Andrew Ross <andrewross@users.sourceforge.net>
##
## DP: Don't run example p7 as part of ctest. It does not work well with postscript
## DP: and also fails on the alpha architecure, for reasons as yet unknown.

@DPATCH@

--- plplot-5.9.5.orig/plplot_test/test_octave.sh.in   2007-10-16 07:57:38.000000000 +0100
+++ plplot-5.9.5/plplot_test/test_octave.sh.in       2007-12-07 15:02:43.000000000 +0000
@@ -54,7 +54,7 @@
 
 # p7 works OK with plmeta, e.g., but not ps or psc. pleop/plbop issue?
 
-	    for i=[1:7 8 9 13 15 16 21] ;
+	    for i=[1:6 8 9 13 15 16 21] ;
 	    if (verbose_test)
 		printf("p%d\n",i);
 		endif
